otelcollector/configmapparser/default-prom-configs/networkobservabilityDefaultDs.yml

```diff
@@ -0,0 +1,87 @@
+scrape_configs:
+  - job_name: "network-observability-cilium"
+    kubernetes_sd_configs:
+      - role: service
+    scheme: http
+    relabel_configs:
+      - source_labels:
+          [
+            __meta_kubernetes_namespace,
+            __meta_kubernetes_service_name,
+            __meta_kubernetes_service_port_name,
+          ]
+        action: keep
+        regex: kube-system;network-observability;cilium
+
+      - source_labels: [__address__]
+        target_label: __address__
+        replacement: $NODE_IP
+        action: replace
+
+      - source_labels: [__address__, __meta_kubernetes_service_port_number]
+        action: replace
+        regex: ([^:]+)(?::\d+)?;(\d+)
+        replacement: $1:$2
+        target_label: __address__
+
+      - source_labels: [__address__]
+        replacement: "$NODE_NAME"
```

Comment:
please make $NODE_IP --> $$NODE_IP$$, and also the same for NODE_NAME (for all 3 jobs)